docs/FAQ.md

```diff
@@ -0,0 +1,12 @@
+# FAQs
+
+## Why balenaSound?
+We wanted to create an open source alternative to existing audio streaming commercial products that was not tied to a vendor or platform. balenaSound is the product of that. balenaSound is free, secure and fully customizable! You can pick the features that suit you and leave those that don't behind, resting asured that your device will keep on working no matter what.
+
+## Is balenaSound free?
+Yes! balenaSound requires you to sign up for a free [balenaCloud](https://dashboard.balena-cloud.com/signup) account. Your first 10 devices are always
+free and full-featured.
+
+## Privacy: is balenaSound secure? Does it need internet access?
+balenaSound does not not depend on any type of cloud service to deliver a great experience. Only the optional Spotify Connect feature requires an internet connection to work. Other than that, your device will be running balenaOS which does communicate with balenaCloud to get application updates. This can be disabled however and won't affect your experience. Both [balenaSound](https://github.com/balenalabs/balena-sound/) and [balenaOS](https://github.com/balena-os) are open source projects, so feel free to check the source code if you want to know more about how they work!
+
```

Comment:
@tmigone : Please separate questions and add below:

## Does it need internet access?
You don't need an internet connection to use balenaSound as speakers (assuming all the apps are downloaded to the devices, set up, and updated), but you need to have music sources on the same shared network or bluetooth-connected devices within range. Accessing apps like Spotify as a music source requires an internet connection.